packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -5,36 +5,21 @@
 // This file is included from `<module>/.android/include_flutter.groovy`,
 // so it can be versioned with the Flutter SDK.
 
-import groovy.json.JsonSlurper
+// TODO: test if it's working
+def pathToThisDirectory = buildscript.sourceFile.parentFile
+apply from: "$pathToThisDirectory/src/main/groovy/shared.groovy"
```

Comment:
Can you import this instead of `apply`ing?

Reply:
Ah, right. I know that feel, trying to make Gradle do something for 2 days.

You can't `import NativePluginLoader` in `module_plugin_loader.gradle` probably because `module_plugin_loader.gradle` they're not in the same Java package. And you didn't set up these directories/packages to work together in Gradle.

I think the best and simplest solution here would be to move `module_plugin_loader.gradle` to `gradle/src/main/groovy`. Then you will not need any `import` at all – the files will be in the same packge. I did something similar with `flutter.groovy` in #123511. I think the same should be done here.

Otherwise Flutter' Gradle plugins will incur even more technical debt.

Reply:
Not possible at the moment, see:
https://github.com/flutter/flutter/pull/137115#discussion_r1390489162